oasislmf/cmd/api.py

```diff
@@ -42,14 +42,32 @@ def load_credentials(login_arg, logger=None):
 
     try:
         api_login = {}
-        api_login['username'] = six.input('Username: ')
+        api_login['username'] = six.moves.input('Username: ')
         api_login['password'] = getpass.getpass('Password: ')
         return api_login
     except KeyboardInterrupt as e:
         logger.info('\nFailed to get API login details:')
         sys.exit(1)
 
 
+def open_api_connection(input_args, logger):
+    if not input_args.get('api_server_url'):
+        logger.info('Error: argument `--api-server-url` not set')
```

Comment:
May be `logger.error` might be more appropriate here. Same elsewhere where you're using `logger.info` to report errors/exceptions.